website/src/components/canvas/Sidebar.jsx

```diff
@@ -89,7 +112,18 @@ class Sidebar extends React.Component {
   mouseEnter() {
     this.setState({opacity: 1})
   }
-
+  onClickClose() {
+   this.setState(prevState => ({
+     isOpen: !prevState.isOpen,
+     closed: 1
+   }));
+ }
+ onClickOpen() {
```

Comment:
How about "OnClickSidebarClose" and "OnClickSidebarOpen"
Ah, that makes sense.
I'll switch it back for consistency sake